README.md

```diff
@@ -886,6 +886,7 @@ A [news aggregator](https://en.wikipedia.org/wiki/News_aggregator), also termed
 - [RSS Fulltext Proxy](https://github.com/Kombustor/rss-fulltext-proxy) - Mirrors RSS feeds to return the full content of the items, extracted from the website. `MIT` `Nodejs`
 - [RSS Monster](https://github.com/pietheinstrengholt/rssmonster) - An easy to use web-based RSS aggregator and reader compatible with the Fever API (alternative to Google Reader). `MIT` `PHP`
 - [RSS2EMail](https://github.com/rss2email/rss2email) - Fetches RSS/Atom-feeds and pushes new Content to any email-receiver, supports OPML. `GPL-2.0` `Python`
+- [RSSHub](https://github.com/DIYgod/RSSHub) - RSSHub is an open source, easy to use, and extensible RSS feed aggregator, it's capable of generating RSS feeds from pretty much everything. ([Demo](https://rsshub.app)) `MIT` `Nodejs`
```

Comment:
> https://docs.rsshub.app/install 

- Docker installation is listed first
- Cleaned up description
- Added main site and moved source code


```suggestion
- [RSSHub](https://docs.rsshub.app/) - An easy to use, and extensible RSS feed aggregator, it's capable of generating RSS feeds from pretty much everything ranging from social media to university departments. ([Demo](https://rsshub.app), [Source Code](https://github.com/DIYgod/RSSHub)) `MIT` `Docker/Nodejs`
```